### DISCUSSION TITLE
IEEE 802.11 Handover - Showcase

### DISCUSSION THREAD

**Discussion by rhornig** [inet-framework/inet#590] [Showcases]
Discussion about the IEEE 802.11 Handover showcase

**Comment by Bhargav2511** [comment]
hello rhornig ,
can you please provide me with source file and ned file for simulating 802.11 handover in omnet++

**Comment by levy** [comment]
You can find the showcase source at https://github.com/inet-framework/inet-showcases/tree/master/wireless/handover and it's documentation is online at https://inet.omnetpp.org/inet-showcases//wireless/handover/

**Reply by ayubshah90** [reply]
Hello!! is there any way to reduce or limit the number of beacons between AP or Between Wireless hosts in the network?
i created an extended handover network and installed a TCP echo App between a standard node and wireless node. but i see there are so many beacons messages between wireless hosts and also the source (wireless host) is sending TCP packets to all other  nodes too (wireless hosts), plus the destination node(standard node) is sending ack message to all nodes in the network

**Comment by ayubshah90** [comment]
why the Source node(wireless host) is sending the TCP packet to all other nodes in the network (wireless hosts)?
i just want a tcp packet to be send to destination and receive an ACK from destination. but i see Source and Destination are sending packets to all other nodes too?

**Comment by levy** [comment]
That's just the physical layer, the link layer will filter out the received packet in all nodes but the destination.

**Reply by ayubshah90** [reply]
okay i must understand it more and will check the result of what you said..
can you please tell me how i can stop from sending and receiving beacons between wireless hosts?
i have multiple AP's and Multiple Wireless Hosts, this thing is understandable that AP sent beacons to Wireless hosts but as the simulation run!! there are so many beacons being sent from all other Wireless Hosts too.
